src/commander/commander_btc.c

```diff
@@ -30,6 +30,9 @@ static const char* _coin_tltc = "LTC Testnet";
 
 commander_error_t commander_btc_pub(const BTCPubRequest* request, PubResponse* response)
 {
+    if (!app_btc_enabled(request->coin)) {
+        return COMMANDER_ERR_DISABLED;
```

Comment:
this line is now unreachable, since you already check in the main commander function.

Imho we should have those checks either all in the main commander func, or in the individual functions, but not both.